<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userInfoDialog</class>
 <widget class="QDialog" name="userInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - UserInfo</string>
  </property>
  <widget class="QLineEdit" name="cnNameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入你的姓名，10个汉字以内</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="enNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>164</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>汉语拼音：</string>
   </property>
  </widget>
  <widget class="QLabel" name="stuIDLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>64</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cnNameLabel">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>114</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>214</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>填好了</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>254</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>24</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请填写你的个人信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="enNameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>16个字母以内，包含空格</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="stuIDLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入12位学号，仅限数字</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
